mapping-for-navigation/_posts/0000-01-11-dual-carriageways.md

```diff
@@ -6,93 +6,101 @@ title: Remodeling Dual-carriageways
 
 *A German dual carriageway in the 1930s ([Wikipedia](https://upload.wikimedia.org/wikipedia/commons/thumb/b/bc/German_Autobahn_1936_1939.jpg/1200px-German_Autobahn_1936_1939.jpg))*
 
-"A divided highway (also separated highway) is any highway where traffic flows are physically separated by a barrier
-(e.g., grass, concrete, steel), which prevents movements between said flows"
-([OpenStreetMap wiki]( https://wiki.openstreetmap.org/wiki/Editing_Standards_and_Conventions#Divided_highways)).
+> A divided highway (also separated highway) is any highway where traffic flows are physically separated by a barrier (e.g., grass, concrete, steel), which prevents movements between said flows.
+>
+> [OpenStreetMap wiki]( https://wiki.openstreetmap.org/wiki/Editing_Standards_and_Conventions#Divided_highways)
+
 We oftentimes encounter cases where a bi-directional highway should be mapped as a dual-carriageway to better
 represent what's on the ground. OpenStreetMap mappers have adopted different approaches to mapping dual-carriageways
-based on the source of data, local community best practices and personal preference.  This guide distills these best
+based on the source of data, local community best practices and personal preference. This guide distills these best
 practices on how to improve road network data in OpenStreetMap by re-mapping bi-directional highways to dual-carriageways.
 
 Our basic rule of thumb are:
 
 *   Keep things simple, add complexity only when necessary.
-*   Dual-carriageway should have a physical barrier visible from imagery or in some cases road marking that
-show its illegal to pass on the other side (MUTCD).
+*   Dual-carriageway should have a physical barrier visible from imagery or in some cases road marking that show its illegal to pass on the other side (MUTCD).
 *   There is enough length from the physical divider (~20 meters?).
-*   During remapping, all other elements associated to the road should be re-tagged (for example,
-turn lanes, turn restrictions, route relations. etc.
+*   During remapping, all other elements associated to the road should be re-tagged (for example, turn lanes, turn restrictions, route relation, etc.)
 
 ## Changing from a simple bi-direction to a dual-carriageway
 
 <!-- https://www.openstreetmap.org/#map=18/37.955426/-121.281075 -->
 
 |Before|After|
 |---|---|
-|![]({{site.baseurl}}/images/dual-carriageway/add-a-way-before.png)|![]({{site.baseurl}}/images/dual-carriageway/add-a-way-after.png)|
+|[![][4ae13b0b]][4ae13b0b]|[![][082fb704]][082fb704]|
```

Comment:
@Andygol while this looks nice, I'm not sure adding complexity to image links is necessary.  Can we just use raw image links rather than link referencing?

Reply:
@maning such link ref helps us to follow DRY (don't repeat yourself) principles.
I added it to provide possibilities to enlarge images for a detailed view.

`[![]({{site.baseurl}}/images/dual-carriageway/add-a-way-before.png)](![]({{site.baseurl}}/images/dual-carriageway/add-a-way-before.png)`

☝️ this full notation of this short ref `[![][4ae13b0b]][4ae13b0b]`

Instead of `4ae13b0b`, you can use something human readable -> `add-a-way-before`

Reply:
Cool, will retain this.